components/collaboration/collaboration-details/CollaborationDetailsContent.tsx

```
@@ -521,81 +546,87 @@ const CollborationDetailsContent = (
+                  chipText={content}
+                  chipIcon={
+                    content === "Instagram"
+                      ? faInstagram
+                      : content === "Facebook"
+                        ? faFacebook
+                        : content === "Youtube"
+                          ? faYoutube
+                          : faInstagram
+                  }
+                />
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)